src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig

```diff
@@ -1,7 +1,17 @@
+{%
+    set icons = {
```

Comment:
This type -> icon map looks duplicated in the templates, I guess we should create a Twig macro for that and use it in these templates.